src/fsharp/FSharp.Core/control.fs

```diff
@@ -729,7 +729,7 @@ namespace Microsoft.FSharp.Control
                     )
                     
         let startThreadWithTrampoline (trampolineHolder:TrampolineHolder) (f : unit -> FakeUnitValue) =
-#if FX_NO_THREADPOOL
+#if FX_THREADPOOL
                 if not (ThreadPool.QueueUserWorkItem((waitCallbackForQueueWorkItemWithTrampoline trampolineHolder), f |> box)) then
```

Comment:
I'm not so sure about that.

The FAKE threadpool is only defined under FSHARP_CORE_NETCORE_PORTABLE.  Which as you correctly point out occurs in the portable library.

The #if FX_NO_THREADPOOL was added to switch between using ThreadPool or new Thread(blah)
This interesting part was that it used ThreadPool when  FX_NO_THREADPOOL was defined. otherwise it creates a new thread and starts it.

It should be noted that for the coreclr we are not defining FSHARP_CORE_NETCORE_PORTABLE it is only defined in profiles 7, 78 and 259.